File: Section Reports/Data/BoundData/C#/BoundData/Invoice.cs
namespace GrapeCity.ActiveReports.Samples.BoundData
{
	public partial class Invoice : GrapeCity.ActiveReports.SectionReport
	{
		public Invoice()
		{
			InitializeComponent();
			Document.Name = "Invoice ActiveReport";
			Data.OleDBDataSource dS = new Data.OleDBDataSource();
			dS.ConnectionString = Properties.Resources.ConnectionString;
			dS.SQL = "SELECT * FROM Invoices ORDER BY Customers.CompanyName, OrderID";
			DataSource = dS;
		}
	}
}

File: Section Reports/Data/BoundData/VB.NET/BoundData/Invoice.vb
Public Class Invoice
	Inherits GrapeCity.ActiveReports.SectionReport
	Public Sub New()
		MyBase.New()
		InitializeComponent()
		Document.Name = "Invoice ActiveReport"
		Dim oDS As New Data.OleDBDataSource
		oDS.ConnectionString = My.Resources.ConnectionString
		oDS.SQL = "SELECT * FROM Invoices ORDER BY Customers.CompanyName, OrderID"
		DataSource = oDS
	End Sub
	
	Public ds As Data.OleDBDataSource
End Class
